Wikipedia/Code/NotificationsCenterCellViewModel.swift

```diff
@@ -47,6 +47,24 @@ final class NotificationsCenterCellViewModel {
         
     }
     
+    func accessibilityText() -> NSAttributedString? {
+        let readAccessibilityText = WMFLocalizedString("notifications-center-cell-read-accessibility-label", value: "Read", comment: "Accessibility text for indicating that a notification's contents have been read.")
+        let unreadAccessibilityText = WMFLocalizedString("notifications-center-cell-unread-accessibility-label", value: "Unread", comment: "Accessibility text for indicating that a notification's contents have not been read.")
+        let readPronounciationAccessibilityAttribute = WMFLocalizedString("notifications-center-cell-read-ipa-accessibility-attribute", value: "rɛd", comment: "Accessibility ipa pronounciation for indicating that a notification's contents have been read.")
```

Comment:
I'm curious, why do we need the phonetic for these two? I'm concerned it won't be translated well.

Reply:
In English VoiceOver reads it as "riːd" instead of "rɛd" which I thought was confusing given the context. 
In our `TalkPageTopicCell` we use the same solution, and in Portuguese, the translation works well. But I'm ok with removing it if it causes issues for any language.

Reply:
I didn't realize this was a strategy we used elsewhere, sorry! In that case, could we re-use the same localization strings? We could move them into `CommonStrings`, and refer to the same one in each spot. 

We could probably update the translation's `comment` to explain that it's now used in a couple places. While it would be good if could update the name of the string, that would cause us to lose all the translations that have been created for this one, so probably best to keep it as `talk-page-discussion-unread-ipa-accessibility-attribute` and `talk-page-discussion-read-ipa-accessibility-attribute`.